gladly_apis/actions/lookup_customer_by_id/response_transformation.gtpl

```diff
@@ -0,0 +1,31 @@
+{{if eq .response.statusCode 200}}
+
+    {{- /* Create new result dict */ -}}
+    {{- $result := dict -}}
+
+    {{- /* Copy all fields from rawData */ -}}
+    {{- range $key, $value := .rawData -}}
```

Comment:
Given my requested changes to the schema, I would recommend just deleting fields that are not wanted from the original `.rawData` and then replacing `customAttributes` with the new structure defined in the schema.